src/styles/styles.js

```diff
@@ -743,11 +741,20 @@ const styles = {
     },
 
     hamburgerOpen: {
-        borderRightWidth: 1,
         borderColor: themeColors.border,
         width: 300,
     },
 
+    w300br1: {
```

Comment:
Could you please refactor this a bit so that `w300` and `w0` are in `src/styles/utilities/sizing.js`, then use them in here? Something like this:

```
w300b1: {
    ...sizing.w300,
    borderRightWidth: 1,
}
```

Also, maybe rename these styles to match their usage a bit more closely? Something like:

`w300br1` -> `hamburgerShown`,
`w0br0` -> `hamburgerHidden`

Reply:
Hey, which one exactly should I get?
@marcaaron or @roryabraham

Reply:
Can we change to `sidebarVisible` and `sidebarHidden` please? We are going to remove the "hamburger" language soon. Thanks! 🍔

Reply:
@marcaaron @roryabraham 
Updated